input_templates/GATKSVPipelineSingleSample.ref_panel_1kg.na12878.json.tmpl

```diff
@@ -1,8 +1,8 @@
 {
-  "GATKSVPipelineSingleSample.sample_id" : "NA12878",
-  "GATKSVPipelineSingleSample.batch" : "test_na12878",
-  "GATKSVPipelineSingleSample.bam_or_cram_file" : "gs://fc-56ac46ea-efc4-4683-b6d5-6d95bed41c5e/CCDG_13607/Project_CCDG_13607_B01_GRM_WGS.cram.2019-02-06/Sample_NA12878/analysis/NA12878.final.cram",
-  "GATKSVPipelineSingleSample.requester_pays_cram": "true",
+  "GATKSVPipelineSingleSample.sample_id" : {{ test_batch.sample_id | tojson }},
```

Comment:
`test_batch` seems like a confusing identifier since it's only one sample. Could this be renamed to `single_sample` or something similar?